### DISCUSSION TITLE
Separate Mobile Notification Settings

### DISCUSSION THREAD

**Discussion by k3vmcd** [element-hq/element-meta#955] [Ideas]
Allow users to set different notification settings for Desktop vs. Mobile per-conversation/room.

(Similar to attached Slack screenshot)

![image](https://cloud.githubusercontent.com/assets/5653317/21462779/350d3abe-c91d-11e6-9c18-9b4ae03795c0.png)

**Comment by ara4n** [comment]
Totally agreed.

**Comment by t3chguy** [comment]
@richvdh might this throw a spanner into the works of your current Notifications re-think?

**Comment by richvdh** [comment]
the way we would support this would be to have different devices use different "notifications profiles": https://docs.google.com/document/d/1s6oGuScNSe4O5uTCC_9lZ7RFF3Ps98u0SI4kAhe4Xoc/edit?disco=AAAAG04c2iY

**Comment by t3chguy** [comment]
@nadonomy given the document you shared this morning, how would this fit in?
